tests/test_frame.py

```diff
@@ -2332,3 +2332,14 @@ def test_getsetitem_multiindex() -> None:
     multi_index: tuple[str, str] = ("Year 1", "Q1")
     budget.loc["project A", multi_index] = 4700
     check(assert_type(budget.loc["project A", multi_index], Scalar), int)
+
+def test_frame_dropna_subset() -> None:
```

Comment:
Can you add a comment here referring to the Github issue, i.e., `# GH 434`

Reply:
added comment